pkg/operator/operator.go

```diff
@@ -346,7 +348,14 @@ func (o *Operator) ensureCerts(ctx context.Context, dir string) ([]byte, error)
 		if err != nil {
 			return nil, errors.Wrap(err, "decoding TLS key")
 		}
-	} else if o.opts.TLSKey == "" && o.opts.TLSCert == "" {
+		caData, err = base64.StdEncoding.DecodeString(o.opts.CACert)
+		if err != nil {
+			return nil, errors.Wrap(err, "decoding certificate authority")
+		}
+		if err = ioutil.WriteFile(filepath.Join(dir, "ca.crt"), caData, 0666); err != nil {
+			return nil, errors.Wrap(err, "create certificate authority file")
+		}
```

Comment:
We don't need to write that file. There's nothing looking at it – we only need to return it to propagate it to the webhook config resource.